src/leo_mq_server.erl

```diff
@@ -373,7 +373,6 @@ terminate(_Reason, #state{id = Id,
                            {line, ?LINE}, {body, Id}]),
     %% Close the backend_db
     MQDBMessageId = MQProps#mq_properties.mqdb_id,
```

Comment:
@mocchira L370 `mq-properties = MQProps`, L374 and L375 are unnecessary rows too.